### ISSUE TITLE
Success Criterion Email 3.3.1 (Error Identification) - Example's Primary Text is Missing

### ISSUE THREAD

**Issue by Grunet** [Grunet/digestible-wcag#5]
Compare what's shown in the email ([link to in-browser version](https://htmlpreview.github.io/?https://raw.githubusercontent.com/Grunet/digestible-wcag-sc-emails/master/dist/3-3-1.html))

![image](https://user-images.githubusercontent.com/29322882/102956100-236f8f00-449d-11eb-963a-095e33a9485a.png)

To what's on [the WCAG understanding page](https://www.w3.org/WAI/WCAG21/Understanding/error-identification.html#examples)

![image](https://user-images.githubusercontent.com/29322882/102956212-587be180-449d-11eb-9c41-aa600d8f7509.png)

**Comment by Grunet** [comment]
This is coming from the examples not having been scraped correctly.

Notice how there are 2 examples in total for this success criteria on [the understanding page](https://www.w3.org/WAI/WCAG21/Understanding/error-identification.html#examples) but 3 end up appearing in [the scraped JSON](https://raw.githubusercontent.com/Grunet/digestible-wcag-scraping/master/dist/wcag22.json) as the 1st example is split in two by the scraping code.

![image](https://user-images.githubusercontent.com/29322882/102957007-f1f7c300-449e-11eb-917b-64b9f7dc3ac8.png)

![image](https://user-images.githubusercontent.com/29322882/102957073-1e134400-449f-11eb-9264-65d5a2e146cd.png)

**Comment by Grunet** [comment]
The specific issue is that [the scraping code](https://github.com/Grunet/digestible-wcag-scraping/blob/master/src/understandingPages/scrape.ts) is getting confused by the div elements within the first example, causing it to think the additional "NOTE" text is actually a separate example.

Here's what the markup of that example from [the examples section](https://www.w3.org/WAI/WCAG21/Understanding/error-identification.html#examples) looks like.

![image](https://user-images.githubusercontent.com/29322882/102957336-cd501b00-449f-11eb-80e0-3c2a4e8046f0.png)

Here's the function (__extractContentWithParagraphs) that's scraping the info in this case. The issue is that the code finds 3 paragraph elements, 2 of which share the same parent and so ultimately output the same content (which gets de-duped). But the 3rd paragraph element is extracted on its own, and outputs different content, resulting in the function outputting 2 pieces of content instead of 1 combination of all 3 elements.

![image](https://user-images.githubusercontent.com/29322882/102957617-7d258880-44a0-11eb-8e18-afd7247a1567.png)

**Comment by Grunet** [comment]
As is done when [scraping content from the main WCAG page](https://github.com/Grunet/digestible-wcag-scraping/blob/master/src/mainDocument/scrape.ts) it might be appropriate to just remove the extra "NOTE" div so that the rest of the example is correctly pulled downstream. At the very least it'd be consistent.

FYI this is the line that does that for scraping the main WCAG page
```html
$(successCrit).find(".note").remove();
```

**Comment by Grunet** [comment]
Also FWIW this issue is very similar in structure to [this earlier issue with the 2.4.3 email](https://github.com/Grunet/digestible-wcag/issues/4), but the underlying root cause is different.

**Comment by Grunet** [comment]
Incidentally addressed this while updating everything to the May 2020 version of WCAG 2.2, where I ended up deciding to get rid of any "Note" section in the examples to handle an issue 3.2.7 was having (matching how the main success criteria text is handled too).

After that, the Flesch-Kincaid metric actually ended up deciding to pick the next example (the one about the user failing to fill in two fields on the form) interestingly enough.
